src/ExpressiveInstaller/Resources/templates/plates-home-page.phtml

```diff
@@ -15,7 +15,7 @@
 <div class="row">
     <div class="col-md-4">
         <h2>
-            <a href="https://zend-expressive.readthedocs.org/en/latest/features/" target="_blank">
+            <a href="https://zendframework.github.io/zend-expressive/features/application/" target="_blank">
```

Comment:
This should likely be to `/getting-started/features/`; I can update that on merge, however.